Marlin/src/inc/Conditionals_post.h

```diff
@@ -469,6 +473,45 @@
   #undef HEATER_1_MAXTEMP
 #endif
 
+#if MAX6675_0_IS_MAX31855 || MAX6675_1_IS_MAX31855
+  #define MAX6675_HAS_MAX31855 1
+  #if PIN_EXISTS(MAX6675_DO) || PIN_EXISTS(MAX31855_MISO)
```

Comment:
Again: why keeping different names for that same PIN? `_DO` and `_MISO` are the same?

Reply:
Please see my comment above

Reply:
We just need a compatibility layer, to avoid a lot of ifs in the code.

Reply:
@thinkyhead has already made changes

Reply:
I fixed this so that  MAX????_DO_PIN is now put into MAX????_MISO_PIN in conditionals_post.h file.  When temperature.cpp  code is run it will use the MAX????_MISO_PIN variable.